ui/imports/shared/stores/RootStore.qml

```diff
@@ -105,9 +106,12 @@ QtObject {
         globalUtils.copyToClipboard(text)
     }
 
-    property var gifColumnA: chatSectionChatContentInputArea.gifColumnA
-    property var gifColumnB: chatSectionChatContentInputArea.gifColumnB
-    property var gifColumnC: chatSectionChatContentInputArea.gifColumnC
+    property var gifColumnA: typeof chatSectionChatContentInputArea !== "undefined" ? chatSectionChatContentInputArea.gifColumnA
```

Comment:
`chatSectionChatContentInputArea` is used in many more places in this file, why don't we use typeof check there as well?

Reply:
fixed :)